### DISCUSSION TITLE
backend.ai run -c 'print("hello world!")' python  명령어 실행시 BackendAPIError: 401 Unauthorize

### DISCUSSION THREAD

**Discussion by eunjungchoi** [lablup/contribution-academy-2021#73] [Q&A]
backend.ai-dev/client-py  디렉터리에서  \
**$ backend.ai run -c 'print("hello world!")' python**    <= 명령어 실행시 에러가 납니다. 

(에러 메시지는 다음과 같습니다. ) 
∙ Session token prefix: pysdk-44fbed3418
∙ In the legacy mode, all cases will run serially!
**✘ BackendAPIError: 401 Unauthorized**
  **Credential/signature mismatch.**

<img width="1267" alt="스크린샷 2021-08-21 오후 10 52 40" src="https://user-images.githubusercontent.com/5889435/130323920-46284ca3-e846-4fb8-97b2-f0fcdf1c30fb.png">

그 전에 다른 부분들은 모두 설치가 완료되어 서버가 잘 동작합니다. 
$ python -m ai.backend.agent.server --debug \
$ python -m ai.backend.manager.server \
$ python -m ai.backend.web.server \
$ python -m ai.backend.storage.server 

(agent)
<img width="1219" alt="스크린샷 2021-08-21 오후 10 59 53" src="https://user-images.githubusercontent.com/5889435/130324113-2bc5ea49-31fe-43ef-87f0-0afc2415e55f.png">
(manager)
<img width="1329" alt="스크린샷 2021-08-21 오후 10 55 29" src="https://user-images.githubusercontent.com/5889435/130324070-53b3dfd0-4dee-42c5-8d02-2bf8289261ac.png">
(webserver)
<img width="1209" alt="스크린샷 2021-08-21 오후 10 56 09" src="https://user-images.githubusercontent.com/5889435/130324072-cd83f65f-1d78-4897-b9ff-0b3fe15dce53.png">
(storage)
<img width="1215" alt="스크린샷 2021-08-21 오후 10 57 21" src="https://user-images.githubusercontent.com/5889435/130324076-ee80779a-0127-47c9-99e8-7f6278b38dbb.png">

**Comment by 1hk-star** [comment]
클라이언트 실행 경로에서 (client-py) source my-backend-ai-session.sh를 한번 입력해줍니다. 기존의 api모드(my-backend-ai.sh)에서 세션 모드로 사용을 하겠다는 의미입니다. 그 후에는 backend.ai login을 통해 발급받은 아이디 비번으로 로그인하시고 다시 실행해주시면 됩니다.

**Reply by eunjungchoi** [reply]
감사합니다. 말씀하신대로 했더니 잘 동작하네요. 🙌
